meta-mender-core/classes/mender-install-ubi.bbclass

```diff
@@ -0,0 +1,25 @@
+ARTIFACTIMG_FSTYPE ?= "ubifs"
+
+MENDER_STORAGE_DEVICE ?= "ubi0"
```

Comment:
Here we're running into a tricky issue, because this will be reused by the Mender client to open the device file, hence this has to be a `/dev` reference. Is the plain `ubi0` reference used anywhere outside of U-Boot? Won't the kernel always refer to a device file, like `/dev/mtd`?

The way I suggest is to stick to `/dev` references in this variable, and use the functions in `mender-helpers.bbclass` to obtain the references needed for U-Boot. But I'm still a bit confused about the different naming, so I might be way off the mark.